src/goto-instrument/cover.cpp

```diff
@@ -28,21 +28,27 @@ Date: May 2016
 void instrument_cover_goals(
   goto_programt &goto_program,
   const cover_instrumenterst &instrumenters,
-  message_handlert &message_handler)
+  message_handlert &message_handler,
+  const irep_idt &mode)
```

Comment:
Doxygen needs adding for the extra parameter, and as a minor nit, I might be tempted to put the new `mode` parameter before the `message_handler` parameter as the diff would be a fraction smaller, and it groups the parameters that have more meaning together i.e. leaving just the UI parameter tacked on the end.